homeassistant/helpers/entity_platform.py

```diff
@@ -346,11 +346,11 @@ async def _async_setup_platform(
 
                 # Block till all entities are done
                 while self._tasks:
-                    pending = [task for task in self._tasks if not task.done()]
```

Comment:
I had to remove the `.done()` check here because it meant we swallowed exceptions because the task was already done